docs/for-developers/resolve-findings-through-comments.md

```diff
@@ -0,0 +1,115 @@
+_**Figure**. GitHub enables you to commit the suggestion from Semgrep directly, fixing the finding._
+
+:::info
+If a line of code contains several findings, Semgrep does not provide the autofix or **Commit suggestion** feature to prevent fixes from conflicting.
+:::
+
+### Semgrep Assistant remediations
+
+Semgrep Assistant provides the following AI-powered security recommendations:
+
+- Step-by-step instructions.
+- AI-written code fixes if a human-written autofix is not available and a code fix can resolve the finding.
+- "Safe to ignore" suggestions.
+
+![PR comment with an AI-written fix.](/img/comment-with-ai-fix.png#md-width)
+_**Figure**. PR comment with an AI-written fix and step-by-step instructions._
+
+![Semgrep Assistant suggesting that a finding is safe to ignore.](/img/ai-assessment-tp-fp.png#md-width)
+_**Figure**. Semgrep Assistant suggesting that a finding is safe to ignore._
+
+## Ignore findings
+
+If the finding is a false positive, acceptable risk, or similar, you can choose to ignore the finding. You can ignore findings directly from your SCM by **replying** to the finding comment. 
+
+1. Find an open comment created by Semgrep AppSec Platform in your pull request or merge request:
+    ![Screenshot of Semgrep AppSec Platform comment in GitHub](/img/semgrep-app-comment-github-beta.png#md-width)
+2. In a subsequent comment, reply with the action you want to take. You must provide a **reason** to help the reader understand why the finding has been triaged as **ignored**:
+
+    | Comment | Description |
+    | - | - |
+    | <code>/fp <span className="placeholder">&lt;REASON&gt;</span></code> | Triage a finding as **Ignored** with the triage reason **false positive**. |
+    | <code>/ar <span className="placeholder">&lt;REASON&gt;</span></code> | Triage a finding as **Ignored** with the triage reason **acceptable risk**. |
+    | <code>/other <span className="placeholder">&lt;REASON&gt;</span></code> | Triage a finding as **Ignored** without specifying the reason; the triage reason value is set to **No triage reason**. |
+    | <code>/open <span className="placeholder">&lt;REASON&gt;</span></code> | Reopen a finding that has been triaged as **Ignored**. The comment is optional. |
+    | <code>/remember <span className="placeholder">&lt;REASON&gt;</span></code> | [Add Assistant Memories](/semgrep-assistant/getting-started#add-memories-beta). |
+
+![A completed triage flow.](/img/pr-comment-triage-response.png#md-width)
+_**Figure**. A completed triage flow._
+
+### Re-run a job or workflow
+
+After resolving or triaging the findings in your PR or MR, you must re-run the Semgrep job or workflow. See the following list for a link to your CI provider's documentation:
+
+- [Re-run a job in GitHub Actions](https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/re-running-workflows-and-jobs)
+- Re-run a job in GitLab CI/CD
```

Comment:
Missing links for these

Reply:
I did not upload my latest commits! Sorry. Fixed.